### ISSUE TITLE
go get does not work anymore (Appengine: http 503)

### ISSUE THREAD

**Issue by olivierlemasle** [rsc/quote#4]
https://rsc.io/quote/v3?go-get=1: returns a HTTP 503 error (URL fetched by `go get `)

```
$ GOPROXY=direct GO111MODULE=on go get rsc.io/quote/v3
go get rsc.io/quote/v3: unrecognized import path "rsc.io/quote/v3": reading https://rsc.io/quote/v3?go-get=1: 503 Service Unavailable
```

Error message:

> Go 1.9 is no longer available. Please refer to https://goo.gl/aESk5L for more information.

**Comment by adamdecaf** [comment]
cc @rsc It looks like this app is running Go 1.9 which has been deprecated.

From https://rsc.io/quote/v3?go-get=1 in a browser: 
> Go 1.9 is no longer available. Please refer to https://goo.gl/aESk5L for more information.

**Comment by dmitshur** [comment]
The immediate issue has been resolved:

```
$ GOPROXY=direct GO111MODULE=on go get rsc.io/quote/v3
go: downloading rsc.io/quote/v3 v3.1.0
go: downloading rsc.io/quote v1.5.2
go: rsc.io/quote/v3 upgrade => v3.1.0
go: downloading rsc.io/sampler v1.3.0
go: downloading golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c
$ echo $?
0
```

There may be follow up steps to take to ensure it doesn't regress, I'll leave it to @rsc to close this issue.

**Comment by rsc** [comment]
Fixed in rsc/swtch@eeaa04271401e89ceee20a7337d38c97d8378bf2, updating to the go114 App Engine runtime.

Before tonight, the last time I had deployed a new version of that app was **Sept 29, 2016**, which would have used **Go 1.6**.

Google Cloud has (it turns out) been emailing me at my personal email address since June 2019, saying that I needed to update to a newer App Engine version or the app would be shut down some time in 2020. I of course did not open any of those emails - I barely keep up with personal mail and rarely even look for mail from companies. 

This is the first time a version change required action on my part. In the past we've always sent those mails and transparently recompiled old apps to move them forward. For example, I successfully ignored the July 31, 2018 mail titled "Please upgrade your App Engine Standard applications to Go version 1.9 by November 1" - App Engine rebuilt it with Go 1.9 for me. The recent transition was more involved and did not include an automatic recompile for old apps, for technical reasons that are clear to me in retrospect (but that I'm not sure are public).

The first mail with a concrete deadline about this transition was sent Feb 5, 2020, saying old apps would shut down on May 31. On May 4, a new mail extended the deadline to June 30. On June 4, another mail reminded that the deadline was June 30. Today, July 28, 2020 the app finally shut down.

The version I uploaded on Sept 29, 2016 thus had an unattended uptime of **3 years, 10 months.** 

I wish it had been longer, but not a bad run.
